EnglishFilter/sections/specific.txt

```diff
@@ -248,6 +248,10 @@ lolhentai.net,vikiporn.com,its.porn,swiftbysundell.com,uiporn.com,fpo.xxx,fapste
 !
 !
 !
+crockotube.com##.right-column-videopage-dif
+crockotube.com##.video.home-bottom.visions
+crockotube.com##.vision-on-player-inside
+crockotube.com##.visions-wrap
```

Comment:
Hi, @Yuki2718

Looks like they use classes with "vision" and "vis" for ads only, I think that it's better to use only these cosmetic rules:

```
crockotube.com##div[class^="vis-"]
crockotube.com##div[class*="vision"]
```

Reply:
Okay, I missed vis- one. Will change soon.